easybuild/easyconfigs/s/Seqmagick/Seqmagick-0.6.1-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2016 University of Geneva Switzerland
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+#
+##
+easyblock = "PythonPackage"
+
+name = 'Seqmagick'
+version = '0.6.1'
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+homepage = 'https://fhcrc.github.io/seqmagick/'
+description = """We often have to convert between sequence formats and do little tasks on them, and it's not worth writing scripts for that. Seqmagick is a kickass little utility built in the spirit of imagemagick to expose the file format conversion in Biopython in a convenient way. Instead of having a big mess of scripts, there is one that takes arguments
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://files.pythonhosted.org/packages/source/s/seqmagick/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+dependencies = [
+    ('Biopython', '1.65', versionsuffix)
+]
+
+sanity_check_paths = {
+    'files': ['bin/seqmagick'],
+    'dirs': ['']
```

Comment:
don't include empty string, either use empty list (`[]`) or specify a particular directory to check for